docs/temporal-explained/timeouts-and-retries.md

```diff
@@ -6,6 +6,8 @@ sidebar_label: Timeouts & retries
 
 import CenteredImage from "../components/CenteredImage.js"
 
+Timeout and retry settings provide fine controls over the behavior of different steps of Workflow Executions and Activity Executions.
```

Comment:
I excised "the behavior of" not because it's inaccurate but to simplify the sentence. Let me know if doing so loses your intent.
```suggestion
Timeout and retry settings provide fine control of various steps of Workflow Executions and Activity Executions.
```
Could also change "fine" to "fine-grained", but I'm inclined to stick with "fine".

Reply:
Hmm - ya i think changing it causes it to be less accurate.